apps/web/src/pages/integrations/UpdateProviderPage.tsx

```diff
@@ -83,6 +84,24 @@ export function UpdateProviderPage() {
     },
   });
 
+  const { mutateAsync: deleteIntegrationApi, isLoading: isDeleting } = useMutation<
+    {},
+    { error: string; message: string; statusCode: number },
+    {
+      id: string;
+      name: string;
+    }
+  >(({ id }) => deleteIntegration(id), {
+    onSuccess: (_, vars) => {
+      successMessage(`Instance configuration ${vars.name} is deleted`);
+      refetch();
+      navigate('/integrations');
```

Comment:
Let's reuse the Routes enum we have in Web: https://github.com/novuhq/novu/blob/8e71c9b09144b5f950c6d8317e2339964051bd93/apps/web/src/constants/routes.enum.ts#L26